_gsocblogs/2022/blog_Acts_TirthankarMazumder.md

```diff
@@ -0,0 +1,129 @@
+---
+project: Acts 
+title: Vectorized Linear Algebra Implementation for Acts
+author: Tirthankar Mazumder
+avatar: https://avatars.githubusercontent.com/u/wermos?s=400&v=4
```

Comment:
This link does not work. To get the correct one, you should go to your GitHub profile and copy the address of your avatar image, using the same size values as you are using in this link. Make sure that the link works in your browser

Reply:
Whoops, my bad. I've fixed it.